src/components/EditListingWizard/EditListingWizard.js

```diff
@@ -27,6 +27,7 @@ import EditListingWizardTab, {
   PHOTOS,
 } from './EditListingWizardTab';
 import css from './EditListingWizard.css';
+import { useEffect } from 'react';
```

Comment:
This could actually be combined with the other import from `react` in the first line

```suggestion
import React, { Component, useEffect } from 'react';
```